ts/features/itwallet/presentation/details/components/ItwPresentationCredentialStatusAlert.tsx

```diff
@@ -28,6 +37,27 @@ const excludedCredentialTypes = [
 
 type ExcludedCredentialTypes = (typeof excludedCredentialTypes)[number];
 
+const LICENSE_RENEWAL_URL = "https://www.mit.gov.it/rinnovo-patente";
+
+type CredentialAlertEvents = "tap_banner" | "open_bottom_sheet" | "press_cta";
+
+export type TrackCredentialAlert = (action: CredentialAlertEvents) => void;
+
+type CredentialStatusAlertProps = {
+  credential: StoredCredential;
+  onTrack: TrackCredentialAlert;
+};
+
+const useAlertPressHandler = (
+  onTrack: TrackCredentialAlert,
+  bottomSheet: { present: () => void }
+) =>
+  useCallback(() => {
```

Comment:
I would remove `useCallback` here, it has a dependency on `onTrack` which is not memoized and the component renders only one time anyway.